input/pages/measure-repository-service.md

```diff
@@ -154,7 +154,7 @@ The package operation supports the ability of a repository to package an artifac
 
 The result of the packaging operation is a Bundle (or Bundles if there is a need to partition based on size) containing the artifact, tailored for content based on the requested capabilities, and any components/dependencies as specified in the parameters.
 
-For the $package operation, when combinations of "in" parameters specify more than one measure to package, the operation SHOULD return an OperationOutcome error. These combinations include the following:
+For the $qm.package operation, when combinations of "in" parameters specify more than one measure to package, the operation SHOULD return an OperationOutcome error. These combinations include the following:
```

Comment:
We've used cqfm as the IG code for the rest of the IG, we should be consistent here and use cqfm